models/gif_cond_exp_multisynapse.cpp

```diff
@@ -143,8 +143,7 @@ nest::gif_cond_exp_multisynapse::Parameters_::Parameters_()
 }
 
 nest::gif_cond_exp_multisynapse::State_::State_( const Parameters_& p )
-  : y_( STATE_VEC_SIZE, 0.0 )
-  , size_neuron_state_( 0 )
+  : y_( STATE_VEC_SIZE + NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR, 0.0 )
```

Comment:
Minor comment: I would shorten this variable name to 'NUM_STATE_ELEMENTS_PER_RECEPTOR' . If you agree, then this should also be changed in 'aeif_cond_alpha_multisynapse' for consistency.

Reply:
It is not the Adaptive Exponential Integrate and Fire model (aeif) but rather the Generalized Leeky Integrate and Fire model (glif or gif)

Reply:
Thanks. I am aware which neuron model you have worked on in this PR. What I meant was: If you change the name of this variable to my proposal in this neuron model (`gif_cond_exp_multisynapse`) (which you did), then we should also change the name accordingly in the other multisynapse neuron models, which are `aeif_cond_alpha_multisynapse` and `aeif_cond_beta_multisynapse`, if I am not mistaken (I simply did  `grep NUMBER_OF_` in the models directory). In `aeif_cond_beta_multisynapse` I am referring to the variable `NUMBER_OF_FIXED_STATES_ELEMENTS`, which could be changed to `NUM_FIXED_STATE_ELEMENTS`.

This could also be done in a separate PR, but since the changes are very small, I would opt for simply adding it this PR, though in a separate commit.

Reply:
Sorry, I misunderstood your first comment. I will do the change in the other models.